src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class Car {
+	private static final int MAX_NAME_LENGTH = 5;
+    private static final int MIN_NAME_LENGTH = 1;
+    private static final String NAME_FORMAT_REGEX = "^[a-zA-Z0-9]*$";
+    private final String name;
+    private int position;
```

Comment:
```suggestion
    private static final int MAX_NAME_LENGTH = 5;
    private static final int MIN_NAME_LENGTH = 1;
    private static final String NAME_FORMAT_REGEX = "^[a-zA-Z0-9]*$";
    
    private final String name;
    private int position;
```

상수랑 필드는 개행하면 더 알기 쉬울 것 같네요!